src/TreeHouse/BehatCommon/PersistenceContextInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace TreeHouse\BehatCommon;
+
+use Behat\Gherkin\Node\TableNode;
+
+interface PersistenceContextInterface
+{
+    /**
+     * @param string    $name
+     * @param TableNode $table
+     */
+    public function theFollowingDataShouldNotHaveBeenPersisted($name, TableNode $table);
```

Comment:
Is deze interface nog wel echt nodig? Hij wordt nu nergens gebruikt, kan me ook niet een use case voorstellen nu.